iot_config/src/lora_field.rs

```diff
@@ -207,13 +207,13 @@ impl From<u32> for LoraField<8> {
 
 impl From<i32> for LoraField<6> {
     fn from(val: i32) -> Self {
-        LoraField::<6>(val as u64)
+        LoraField::<6>((val as u32) as u64)
```

Comment:
Indeed; we only use i32 b/c Postgres doesn’t support u32, but since these numbers overflow the i32 range we have to flip them back to unsigned before changing the scale to a u64

Reply:
Perhaps this would be better expressed as 
```rust
val & (u32::MAX as u64)
```
? 
I find the double cast a little unpleasant to read

Reply:
you'd still have to do multiple casts; there's no impl of `&` for combining a u32 and an i32

Reply:
ooooh cause it's i32, my mistake

Reply:
yeah, thanks Postgres